### PR TITLE
Allow linking to schema tables without osquery version

### LINKED ISSUE #51: Links to tables in schema must specify version
The osquery docs include various references to table schemas of the form https://osquery.io/schema/#file. These links no longer work. Instead, they would need to be replaced with links of the form https://osquery.io/schema/2.11.2#file. With that, they would quickly become outdated.

Maybe we can make the existing links work? If not, maybe https://osquery.io/schema/current#file?

### PR DESCRIPTION
**Pull request by zwass**
Fixes #51

### CODE REVIEW

**Comment by terracatta** [comment]
approved!

File: src/Router/Router.js [hunk 1]
```diff
@@ -19,62 +19,61 @@ class Router extends Component {
...
-              to={`${process.env.PUBLIC_URL}/schema/${currentOsqueryVersion}`}
+            <Route
+              from={`${process.env.PUBLIC_URL}/schema/`}
+              render={({ location }) => (
+                // This render function is a redirect that preserves the search
+                // and hash params. This allows linking directly to tables
+                // without knowing the version number in advance.
+                <Redirect
+                  to={{
+                    pathname: `${process.env.PUBLIC_URL}/schema/${currentOsqueryVersion}/`,
```
Comment (Smjert) [inline src/Router/Router.js:70]: I think the last slash has to be dropped here, just tested on the live website and the 
`https://osquery.io/schema/#ntfs_journal_events` URL gets redirected to `https://osquery.io/schema/4.2.0/#ntfs_journal_events`.

Comment (zwass) [reply src/Router/Router.js:70]: @Smjert is that causing issues for you? All three behave URLs properly for me.

Comment (Smjert) [reply src/Router/Router.js:70]: Sorry it works, should've tested with an incognito session, it was a caching issue ^^'.

**Pull request merged**